### PR TITLE
16.0 studio lpe

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
robodoo rebase-ff

File: addons/web/static/src/views/form/form_compiler.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ export class FormCompiler extends ViewCompiler {
             ? `d-flex {{ uiService.size < ${SIZES.XXL} ? "flex-column" : "flex-nowrap h-100" }}`
             : "d-block";
         const stateClasses =
-            "{{ props.record.isVirtual or props.record.isDirty ? 'o_form_dirty' : 'o_form_saved' }}";
+            "{{ props.record.isDirty ? 'o_form_dirty' : !props.record.isVirtual ? 'o_form_saved' : '' }}";
```
Comment (mcm-odoo) [inline addons/web/static/src/views/form/form_compiler.js:228]: ok for me

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff +

Excellent work everyone :+1:

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**